src/core/ext/transport/binder/wire_format/binder_constants.h

```diff
@@ -19,7 +19,7 @@
 
 #include "absl/base/attributes.h"
 
-#if defined(ANDROID) || defined(__ANDROID__)
+#ifdef GPR_SUPPORT_BINDER_TRANSPORT
```

Comment:
move up to after port_platform.h

Reply:
`#include "absl/base/attributes.h"` is required by both situation below (`ABSL_CONST_INIT`), whether `GPR_SUPPORT_BINDER_TRANSPORT` is defined or not